docker/Dockerfile.el7

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
```

Comment:
In general, could the directory structure be:

 `images/`
 `containers/`

With likely the top one being more accurate given you are defining the images and a container is an instance of an image.